core/blockly.js

```diff
@@ -62,7 +63,6 @@ goog.require('Blockly.inject');
 goog.require('Blockly.utils');
 goog.require('goog.color');
 
-
```

Comment:
I think the convention in the rest of the files is to have 2 newlines separating the 'goog.require' statements from the code.

Reply:
👍 no need to introduce unnecessary changes.